model01/Makefile

```diff
@@ -6,16 +6,20 @@ deps:
 		cd $(HOME)/Arduino/hardware/keyboardio/avr && make update-submodules; \
 	else \
 		curl https://downloads.arduino.cc/arduino-1.8.5-linux64.tar.xz -o arduino.tar.xz && \
-		tar -xvf arduino.tar.xz -C arduino && \
+		mkdir arduino && \
+		tar -xvf arduino.tar.xz && \
```

Comment:
Will this actually decompress the `tar.gz` into the `arduino` directory? Won't it just put it in the directory above?

Reply:
Yeah, but each non-tarbomb file actually has a subdirectory at the root which is equivalent to its name.